contrib/statesync.bash

```diff
@@ -1,33 +1,48 @@
 #!/bin/bash
 # microtick and bitcanna contributed significantly here.
+# Pebbledb state sync script.
+# invoke like: bash scripts/ss.bash
+
+## USAGE RUNDOWN
+# Not for use on live nodes
+# For use when testing.
+# Assumes that ~/.evmosd doesn't exist
+# can be modified to suit your purposes if ~/.evmosd does already exist
+
 set -uxe
 
 # Set Golang environment variables.
 export GOPATH=~/go
```

Comment:
I would check whether the GOPATH is already set. If not advise the user to set it and add it to that path. 
I use Homebrew and that installs golang to a totally different bin directory.